test/rules/prefer_asserts_in_initializer_list.dart

```diff
@@ -48,13 +51,48 @@ class A {
   // no lint if method is call on other objet
```

Comment:
"objet" --> "object"

This comment says "no lint", but the lines below are marked with "LINT". Seems inconsistent.